index.html

```diff
@@ -6,47 +6,174 @@
+        when an unknown printer took a dummy text.
+    </p>
+    <div class="grid">
+    <ul class="grid-container">
+        <li class="grid-item">css</li>
+        <li class="grid-item">html</li>
+        <li class="grid-item">bootstrap</li>
+        <li class="grid-item">Ruby</li>
+    </ul>
+    </div>
+    <button id="button" type="submit">See Project</button>
+    <br>
+    <div class="bottom">
+    <a href="#"><img src="image.png" alt=""><div class="centered">
+        <h3>Professional Art <br>printing Data</h3>A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard.<br><ul class="grid-containers"><li class="display">Html</li>
+        <li class="display">bootstrap</li>
+        <li class="display">Ruby</li></ul></div></a><button class="btn active" type="submit">See Project</button></div>
+
+    <div class="bottom">
+    <a href="#"><img src="image.png" alt=""> <div class="centered">
+        <h3>Professional Art <br>printing Data</h3>A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard.<br><ul class="grid-containers"> <li class="display">Html</li>
+        <li class="display">bootstrap</li>
+        <li class="display">Ruby</li></ul></div></a><button class="btn" type="submit">See Project</button></div>
+
+<div class="bottom">
+    <a href="#"><img src="image.png" alt=""> <div class="centered">
+        <h3>Professional Art <br>printing Data</h3>A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard.<br><ul class="grid-containers"> <li class="display">Html</li>
+        <li class="display">bootstrap</li>
+        <li class="display">Ruby</li></ul></div></a><button class="btn" type="submit">
+        See Project</button></div>
+
+        <div class="bottom">
+    <a href="#"><img src="image.png" alt=""> <div class="centered">
+        <h3>Professional Art <br>printing Data</h3>A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard.<br><ul class="grid-containers"> <li class="display">Html</li>
+        <li class="display">bootstrap</li>
+        <li class="display">Ruby</li></ul></div></a><button class="btn" type="submit">
+        See Project</button></div>
+    </section>
+    <br>
+    <section class="about">
+    <h2>About Me</h2>
+    <p>
+        Hello I'm a Software Developer! I can help <br>
+        you build a product, feature or website. <br>
+        Look through some of my work and <br> Experience.
+        If you like what you see and<br> have a project dont <br>
+        hesitate to contact me.
+    </p>
+    <button id="Resume" type="submit">Get My Resume</button>
+
+    <div id="ground"></div>
+
+    <hr class="solid" />
+
+    <div class="languages"><img src="img_Rectangle.png" alt="rectangle">
```

Comment:
- [ ] Since this image is used as background I think it could be a good idea to move this `<img>` to your CSS file. Please, address this use for better practices :wink:

Reply:
Thank you so much. Resolved. It is more like a bullet point and not a background image. I appreciate your review.